config/jobs/kubernetes/release/release-config.yaml

```
@@ -38,8 +38,31 @@ presubmits:
       testgrid-tab-name: release-cluster-up
       testgrid-alert-email: release-managers@kubernetes.io
       testgrid-num-columns-recent: '30'
+  - name: pull-release-test
+    alwasy_run: true
+    decorate: true
+    skip_report: true
+    path_alias: k8s.io/release
+    extra_refs:
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+    spec:
+      containers:
+      - image: launcher.gcr.io/google/bazel:0.29.1
+        command:
+        - ../test-infra/hack/bazel.sh
+        args:
+        - test
+        - //...
+    annotations:
+      testgrid-dashboards: sig-release-misc
+      testgrid-tab-name: release-unit
```

```suggestion
      testgrid-tab-name: release-test
```

This currently overlaps w/ the `release-unit` name on the `pull-release-unit`.

---

Done